<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family: 'Open Sans', sans-serif;
font-size:23px;
}

QWidget{
	
	
	background-image: url(:/images/PyProject/UI/images/nightsky.png);
}
QFrame{
background:#1e2833;
border-radius:15px;
}
#RegisterButton{
background:#1df2ef;
border-radius:15px;
}
#RegisterButton:hover{
background:#fffff2;
}
#pushButton_2{
background:#1df2ef;
border-radius:15px;
}
#pushButton_2:hover{
background:#fffff2;
}

#pushButton_3{
background:#DB4437;
border-radius:15px;
}
#pushButton_3:hover{
background:#fffff2;
}
#toolButton{
background:#1df2ef;
color:white;
border-radius:60px;
}
#toolButton:hover{
background:#fffff2;
}
QLabel{
color:#fffff2;
}
QPushButton:hover{
background: #fffff2;
letter-spacing: 1px;

}
QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;

}
#label_4 ,#label_5 , #label_6 ,#label_7{
font-size:18px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>450</width>
      <height>531</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RegisterButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>470</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="UsernameText">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>User Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PasswordText">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ConfirmPasswordText">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>410</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Confirm Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NameText">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/register.jpg</normaloff>images/register.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="BackBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:#1df2ef;
border-radius:  20px ;
}

QPushButton:hover{
background: #fffff2;
letter-spacing: 1px;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/back.png</normaloff>images/back.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources1.qrc"/>
 </resources>
 <connections/>
</ui>
